torchaudio/backend/sox_io_backend.py

```diff
@@ -73,9 +73,12 @@ def load(
 
             * 32-bit floating-point
             * 32-bit signed integer
+            * 24-bit signed integer
             * 16-bit signed integer
             * 8-bit unsigned integer (WAV only)
 
+            Note: 24-bit signed integers are converted to ``int32`` tensors.
```

Comment:
I think the fact that `int32` will be used as a substitute is better covered by the next block of docstring, so let's not mention it here. (Besides, note section inside of note section would not render well https://184250-90321822-gh.circle-artifacts.com/0/docs/backend.html#load)